src/platforms/java/migration.mdx

```diff
@@ -42,11 +42,7 @@ val hints = mutableMapOf<String, Any>("myHint" to "myStringHint")
 Sentry.captureException(RuntimeException("exception"), hints)
 ```
 
-* `SentryOptions#sendClientReports` is now enabled by default. To disable it, use the code snippet below:
-
-```properties
-send-client-reports=false
-```
+* `SentryOptions#sendClientReports` is now enabled by default. To disable it, see [sendClientReports](/platforms/java/configuration/options/#send-client-reports)
```

Comment:
Is this link correct? Worked for me locally.

Reply:
You can test all those sublinks with the preview deployed by vercel.

Reply:
Works :-)